script.js

```diff
@@ -37,11 +108,27 @@ function generateTotalCost(domains) {
     totalCost +
     '</span><br/>on domains every year. ' +
     'That\'s enough to buy<br/><span id="total-num-bobas">' +
-    costToBobas(totalCost).toFixed(1) +
-    '</span><br/>bobas.';
+    costToBobas(
+      calcDomainsCost(domains),
+      flavorPrice,
+      sizePrice,
+      topping1Price,
+      topping2Price
+    ).toFixed(1) +
+    '</span><br/>';
+  totalCostText.innerHTML += size + ' ' + flavor + ' bobas';
+  if (topping1 != 'none') totalCostText.innerHTML += ' with ' + topping1;
+  if (topping2 != 'none') totalCostText.innerHTML += ' and ' + topping2;
   return totalCostText;
 }
 
+/**
```

Comment:
Assuming this is TODO?

Reply:
Oops I was going to make a function to describe the boba order but I found it easier to just do it in the innerHTML directly since we only do it once

Reply:
ty, I will remove that now